recipe/testhdf5exc.cpp

```diff
@@ -0,0 +1,28 @@
+
+#include <iostream>
+#include <H5Cpp.h>
+
+int main()
+{
+    H5::Exception::dontPrint();
+    
+    H5::H5File *keaImgH5File = new H5::H5File("/tmp/test.hd5", H5F_ACC_TRUNC, H5::FileCreatPropList::DEFAULT);
```

Comment:
@gillins, probably name this something like h5file because the reference to keaImg is confusing.